src/adapters/stargate/products/vesting/stargateVestingAdapter.ts

```diff
@@ -89,6 +89,10 @@ export class StargateVestingAdapter
       blockTag: blockNumber,
     })
 
+    if (!userBalance && !lockedAmount) {
```

Comment:
I'm pretty sure that one of them should be dominant, but just in case I'm checking both.